jekyll/_cci2/test-config-policies.adoc

```diff
@@ -0,0 +1,607 @@
+
+. Update the file structure:
++
+[source,shell]
+----
+├── config-policies/
+│   ├── docker/
+│   │   ├── docker.rego
+│   │   ├── docker_test.rego
+│   ├──version/
+│   │   ├── version.rego
+│   │   ├── version_test.rego
+----
+
+. Run all all tests including those in subfolders by appending `/...` to the test path:
++
+[source,shell]
+----
+circleci policy test ./config-policies/...
+----
++
+Expected output. Tests are passing again:
++
+[source,shell]
+----
+?     config-policies            no tests
+ok    config-policies/docker     0.000s
+ok    config-policies/version    0.000s
+
+7/7 tests passed (0.001s)
+----
+
+. To build more confidence, best practice is to create a top level test that will use the entire policy bundle, similar to an integration or end-to-end test.
+. Create a new test file: ./config-policies/policy_test.yaml`
+. Copy the following into your `policy_test.yaml` file:
++
+[source,yaml]
+----
+test_policy:
+  input:
+    version: 2.1
+    jobs:
+      example:
+        steps:
+          - setup_remote_docker:
+              version: 20.10.11
+  decision: &root_decision
+    status: PASS
+    enabled_rules:
+      - check_min_remote_docker_version
+      - check_version
+  cases:
+    bad_remote_docker:
+      input:
+        jobs:
+          example:
+            steps:
+              - setup_remote_docker:
+                  version: 1.0.0
+      decision:
+        <<: *root_decision
+        status: HARD_FAIL
+        hard_failures:
+          - rule: check_min_remote_docker_version
+            reason: 'job "example": remote docker version "1.0.0" is less than minimum required "20.10.11"'
+
+    bad_version:
+      input:
+        version: 1.0
+      decision:
+        <<: *root_decision
+        status: HARD_FAIL
+        hard_failures:
+          - rule: check_version
+            reason: version must be at least 2.1 but got 1
+
+test_break_all_rules:
+  input:
+    version: 1.0
+    jobs:
+      example:
+        steps:
+          - setup_remote_docker:
+              version: 20.0.0
+  decision:
+    <<: *root_decision
+    status: HARD_FAIL
+    hard_failures:
+      - rule: check_min_remote_docker_version
+        reason: 'job "example": remote docker version "20.0.0" is less than minimum required "20.10.11"'
+      - rule: check_version
+        reason: version must be at least 2.1 but got 1
+----
+
+. Run the full set of tests again in verbose mode:
++
+[source,shell]
+----
+circleci policy test ./config-policies/...
+----
++
+Expected output:
++
+[source,shell]
+----
+ok    config-policies            0.001s
+ok    config-policies/docker     0.001s
+ok    config-policies/version    0.001s
+
+11/11 tests passed (0.003s)
+----
+
+## Use metadata with tests
```

Comment:
@davidmdm I'm having trouble with this section, just getting lots of errors when running tests. Could you take a look?